catalogs/bundled-catalog/src/main/java/com/datastrato/gravitino/catalog/property/PropertyConverter.java

```diff
@@ -81,30 +72,14 @@ public Map<String, Object> engineToGravitinoProperties(Map<String, Object> engin
     Map<String, Object> gravitinoProperties = new HashMap<>();
     Map<String, String> engineToGravitinoMapping = engineToGravitinoMapping();
 
-    Map<String, PropertyEntry<?>> propertyEntryMap = gravitinoPropertyMeta();
     for (Map.Entry<String, Object> entry : engineProperties.entrySet()) {
       String gravitinoKey = engineToGravitinoMapping.get(entry.getKey());
       if (gravitinoKey != null) {
-        PropertyEntry<?> propertyEntry = propertyEntryMap.get(gravitinoKey);
```

Comment:
I think the reason you gave is not very persuasive `seems not to have much usage, it use PropertyEntry to decode to an object, it will be transformed to string.`

We use `propertyEntry.decode` to check whether the property value passed to the engine is valid or not, so why do you think it's not very helpful here?

Reply:
could you provide an example from end to end to show the usage?

Reply:
As dissussed with @yuqi1129 , the main usage of the function is to check the type of parameters, if we removed this function, Gravitino server could check the parameter type too which is acceptable.